### PR TITLE
happiness topic

### PR DESCRIPTION
**Pull request by HalHarrison**
Completely new to this so check close that its ok. Enthusiastic to write more!

### CODE REVIEW

**Comment by TacticalTriangle** [comment]
Sad is a keyword for another topic.

**Comment by HalHarrison** [comment]
Good now?

**Comment by TacticalTriangle** [comment]
The keywords check out. I don't know about the writing though. I can't write that well.

File: Monika After Story/game/script-topics.rpy [hunk 1]
```diff
@@ -4790,6 +4790,33 @@ label monika_urgent:
     m 1j "I wish I could see the look on your face right now!"
     m 1k "I'm sure it's adorable~"
     return
+    
+    init 5 python:
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: Remove indentattion here. `init` python blocks are **not** indented.

File: Monika After Story/game/script-topics.rpy [hunk 2]
```diff
@@ -4790,6 +4790,33 @@ label monika_urgent:
...
+    #This is a list of keywords for this topic
+    for key in ['happy' , 'mood']:
+        monika_topics.setdefault(key,[])
+        monika_topics[key].append('monika_happiness') #id
+    monika_random_topics.append('monika_happiness')
+
+label monika_hapiness:
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: your label here is spelled incorrectly.

**Comment by HalHarrison** [comment]
Fixed.
Btw- in the future would it be best for me to submit things one topic at a time or as many topics as I can get done?

**Comment by ThePotatoGuy** [comment]
I recommend multiple at a time. It makes merging pull requests easier. However, I don't do dialogue reviews, so let's see what @monikLover has to say about that.

**Comment by Kiosho** [comment]
m 1p "After all, without you [player] my life would be completely unbearable."
I'd surround [player] with commas here.

m 1h "You may get nervous about your future or about doing well in important social situations~"
m 1e "But always remember to put things into perspective and be grateful for how safe your life really is."
Typically we use "~" to continue a thought into the next line.

And heads up, line 4813 runs off the screen. It may be too long for the dialogue box.

Good topic, just needs tweaks.

**Comment by HalHarrison** [comment]
Made changes
Also line 4513 works in the game and its even longer so 4813 should work. I think?

**Comment by monikLover** [comment]
Edited and rewritten. Approved.

**Pull request merged**